config/config.go

```diff
@@ -35,6 +36,8 @@ type Provider interface {
 // ReadDefaultConfig reads the configuration file
 func ReadDefaultConfig(appName string, configName string) {
 	viper.SetEnvPrefix(appName)
+    replacer := strings.NewReplacer(".", "_")

```

Comment:
I suppose you tried this option. Maybe also add it as a command line ? See if it conflicts with https://github.com/slurdge/goeland/blob/c25bf2960f9e5b3e71c7ddd678614842e937d24f/cmd/root.go#L68

Reply:
Yes, tried it. It works well 👍🏽 
It would be the only email configuration settable with a flag, isn't it confusing? I feel like you would either want to be able to set everything (host, port, username and password) with the command line or nothing but I don't really see a usecase for a mix of both. I'm okay with adding the flag if you think it's the right thing to do 👍🏽

Reply:
Great. I agree that we should expose everything or nothing. For now, let's **not** add it and keep it in the configuration file.